kernel/core/strings/stringfunctions.hpp

```diff
@@ -2791,9 +2825,11 @@ static std::map<std::string, StringFuncHandle> getStringFuncHandles()
     mHandleTable["and"]                 = StringFuncHandle(VAL, strfnc_and, true);
     mHandleTable["ascii"]               = StringFuncHandle(STR, strfnc_ascii, false);
     mHandleTable["basetodec"]           = StringFuncHandle(STR_STR, strfnc_basetodec, false);
+    mHandleTable["beginswith"]          = StringFuncHandle(STR_STR, strfnc_beginswith, false);

```

Comment:
Could probably be removed, can't it?

Reply:
I realized that in the original requirement, #21 asks for beginswith() as the main, and startswith() as a possible alias(), so I left the beginswith() (well, added it back in)

Reply:
`startswith()` is enough. `beginswith()` does not add any reasonable benefit here.

Reply:
Done